### PR TITLE
Add Gnaa Xarr'rr 

### PR DESCRIPTION
**Pull request by mmarquee**
Add Gnaa Xarr'rr (as from Gurps Traveller Aliens 2),

### CODE REVIEW

File: res/Sectors/Gzirr!k'l.xml [hunk 1]
```diff
@@ -38,8 +38,8 @@
...
-<Border WrapLabel="true" Allegiance="HvFd" LabelPosition="2939">3020 3019 3018 3017 3117 3217 3317 3318 3319 3320 3321 3322 3323 3324 3325 3326 3327 3328 3329 3330 3331 3332 3333 3334 3335 3336 3337 3338 3339 3340 3341 3241 3141 3041 2941 2841 2741 2641 2541 2441 2440 2540 2539 2538 2537 2536 2535 2534 2533 2532 2532 2631 2630 2629 2628 2528 2527 2526 2525 2625 2725 2825 2925 2924 2923 3022 3021</Border>
+<Border WrapLabel="false" Color="Green" Allegiance="Kk" LabelPosition="1109">0004 0104 0203 0303 0402 0502 0501 0500 0600 0700 0800 0900 1000 1100 1200 1300 1400 1500 1600 1700 1800 1900 2000 2100 2200 2300 2400 2500 2600 2700 2800 2900 3000 3100 3200 3300 3301 3302 3303 3304 3305 3306 3307 3308 3309 3310 3210 3111 3010 2910 2810 2811 2812 2813 2714 2715 2716 2616 2517 2518 2519 2419 2418 2318 2218 2119 2120 2121 2021 1922 1822 1723 1623 1523 1422 1322 1222 1122 1022 0923 0823 0724 0624 0525 0425 0326 0225 0126 0026 0027 0026 0025 0024 0023 0022 0021 0020 0019 0018 0017 0016 0015 0014 0013 0012 0011 0010 0009 0008 0007 0006 0005 0004</Border>
```
Comment (inexorabletash) [inline res/Sectors/Gzirr!k'l.xml]: Colors shouldn't be necessary - they're applied automatically by a stylesheet based on allegiance.

Unless you're not seeing them somewhere, in which case it's a bug - let me know!

**Comment by inexorabletash** [comment]
Thanks for the PR! Can you point out where the colors aren't showing up? The styles file should make known allegiances the appropriate colors automatically:

https://github.com/inexorabletash/travellermap/blob/master/res/styles/otu.css

File: res/Sectors/Gzirr!k'l.xml (hunk 1 above)
Comment (mmarquee) [inline res/Sectors/Gzirr!k'l.xml]: It maybe that I have an old project, but the borders without colours were showing up as red. I'll remove them and resubmit

**Comment by mmarquee** [comment]
OK, that latest commit should remove the overridden colors.

File: res/Sectors/Gzirr!k'l.xml [hunk 2]
```diff
@@ -39,7 +39,7 @@
   </Allegiances>
   <Borders>
 <Border WrapLabel="false" Allegiance="Kk" LabelPosition="1109">0004 0104 0203 0303 0402 0502 0501 0500 0600 0700 0800 0900 1000 1100 1200 1300 1400 1500 1600 1700 1800 1900 2000 2100 2200 2300 2400 2500 2600 2700 2800 2900 3000 3100 3200 3300 3301 3302 3303 3304 3305 3306 3307 3308 3309 3310 3210 3111 3010 2910 2810 2811 2812 2813 2714 2715 2716 2616 2517 2518 2519 2419 2418 2318 2218 2119 2120 2121 2021 1922 1822 1723 1623 1523 1422 1322 1222 1122 1022 0923 0823 0724 0624 0525 0425 0326 0225 0126 0026 0027 0026 0025 0024 0023 0022 0021 0020 0019 0018 0017 0016 0015 0014 0013 0012 0011 0010 0009 0008 0007 0006 0005 0004</Border>
-<Border WrapLabel="true" Allegiance="HvFd" LabelPosition="2939">3020 3019 3018 3017 3117 3217 3317 3318 3319 3320 3321 3322 3323 3324 3325 3326 3327 3328 3329 3330 3331 3332 3333 3334 3335 3336 3337 3338 3339 3340 3341 3241 3141 3041 2941 2841 2741 2641 2541 2441 2440 2540 2539 2538 2537 2536 2535 2534 2533 2532 2532 2631 2630 2629 2628 2528 2527 2526 2525 2625 2725 2825 2925 2924 2923 3022 3021</Border>
```
Comment (inexorabletash) [inline res/Sectors/Gzirr!k'l.xml]: Allegiance should stay HvFd since we have 4-char codes for those. But I'll merge and fix.

**Pull request merged**